internal/server/v2/resolve/golden/resolve_description/city.json

```diff
@@ -20,6 +20,7 @@
     {
       "node": "New York City",
       "resolvedIds": [
+        "geoId/3651000",
```

Comment:
@Spaceenter do you know why this is?

Reply:
Not sure - but I can debug in a separate PR.